README.en.md

```diff
@@ -0,0 +1,128 @@
+[🇫🇷 Français](README.md)
+
+# COVID19 epidemic french national data
+
+Official information on the progression of the epidemic in France is quite fragmented, and is almost never structured in the form of data.
+
+The objective of this repository is to consolidate official information, and make it available in open and easily reusable formats (JSON, CSV…).
+
+It's useless to waste time writing scrappers, at this stage it is more efficient to copy the data by hand, and to indicate the source.
+
+## Output data files
+
+- [chiffres-cles.json](https://github.com/opencovid19-fr/data/raw/master/dist/chiffres-cles.json)
+- [chiffres-cles.csv](https://github.com/opencovid19-fr/data/raw/master/dist/chiffres-cles.csv)
+
+## Sources used
+
+- [Santé publique France](https://www.santepubliquefrance.fr)
+
+- Regional Health Agencies - Please take the open issues to treat ARS x date, and that this work is not done repeatedly.
```

Comment:
- Regional Health Agencies - Please raise an open issue before handling ARS x date, so that the work is not done repeatedly.